index.html

```diff
@@ -18,200 +18,259 @@
+                          </ul>
+                          <ul>
+                              <li>
+                                  Bootstrap
+                              </li>
+                              <li>Ruby on Rails</li>
+                              <li>RSPec</li>
+                              <li>CapyBara</li>
+                              <li>Selenium</li>
+                          </ul>
+                      </div>
+                      <div class="knowledge">
+                          <h3 class="ski">Skills</h3>
+                          <ul>
+                              <li>Codekit</li>
+                              <li>GitHub</li>
+                              <li>Codepen</li>
+                              <li>Gitlab</li>
+                              <li>Terminal</li>
+                          </ul>
+                      </div>
+                  </div>
+              </section>
+          </div>
+          </div>
+      </div>
+      <div class="horizontal-line-1">
+          <div class="horizontal-1"></div>
+      </div>
+      <div class="image-arrow-1">
+        <div class="image-arrow">
+            <img src="assets/images/arrow-down.svg" alt="" />
+            <h3 class="work">Work</h3>
         </div>
-        <div class="lang">
-            <ul>
-                <li>JavaScript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-            </ul>
-            <ul>
-                <li>
-                    Bootstrap
-                </li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-            </ul>
+        <div class="social-media">
+            <a href="/" class="social__icons--link" target="_blank">
+                <i class="fab fa-arrows"></i>
+            </a>
+            <a href="/" class="social__icons--link" target="_blank">
+                <i class="fab fa-instagram"></i>
+            </a>
+            <a href="/" class="social__icons--link" target="_blank">
+                <i class="fab fa-twitter"></i>
+            </a>
+            <a href="/" class="social__icons--link" target="_blank">
+                <i class="fab fa-linkedin"></i>
+            </a>
+            <a href="/" class="social__icons--link" target="_blank">
+                <i class="fab fa-github"></i>
+            </a>
         </div>
-        <div class="knowledge">
-            <h3 class="ski">Skills</h3>
-            <ul>
-                <li>Codekit</li>
-                <li>GitHub</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
+        <div class="image-arrow-2">
+              <img src="assets/images/arrow-down.svg" alt="" />
+              <h3 class="work">Work</h3>
         </div>
-    </section>
+      </div>
+      <h2 class="recent-work">MY RECENT WORK</h2>
```

Comment:
- [ ] Nice work you have done with your My Works Section, kindly align the rows, so they align with the Figma Template